.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -6,44 +6,52 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for reporting an issue.
-        If you think we can make Dgraph better, describe your feature request here.
-        Note: Feature requests are judged based on user experience and modeled on 
-        [Go Experience Reports](https://github.com/golang/go/wiki/ExperienceReports). 
-        These reports should focus on the problems - they should not focus on and need not propose solutions.
-  - type: dropdown
+      You have an idea for making Ratel better? We'd love to hear about it.
+      Please search the issue list first as there is a chance that someone else had the same idea.
+      Feature requests are tagged `kind/feature`.
+      If you find a similar request, add a thumbs-up to vote for it and optionally add a comment to be part of the conversation.
+
+  - type: textarea
     attributes:
-      label: Have you tried Dgraph before this proposal? and did not find anything similar?
-      options:
-        - "Yes"
-        - "No"
+      label: Use case
+      description: Describe the context in which the feature will be used and what is achieved when the feature is used.
+      placeholder: |
+          Type here.
     validations:
-      required: false
+      required: true
   - type: textarea
     attributes:
-      label: What you wanted to do.
```

Comment:
I'm not sure about removing these step questions.

What you wanted to do.
What you actually did.
Why wasn't it great, with examples.

have you discuss with someone about this?

Reply:
I checked feature request templates on other github repos ( I found nice guides in RStudio repo). Feature request are usually for something new or not possible. So the current questions does not seem to make much sense in that context IMO (more for Bug report). 
But I also created the PR to get engineering feedback on what they need in a Feature Request.